api/v1/swagger.yaml

```diff
@@ -4470,11 +4470,16 @@ paths:
   /token:
     x-swagger-router-controller: token
     post:
-      summary: Create an api access token
+      security:
+        - jwt: []
+      summary: Create a new api access token
       tags: [ token ]
       description: >-
-        Authenticate a user with email and password and create an api access
-        token.
+        Create a new api access token by providing a token in header.
+        If the Refocus configuration parameter `useAccessToken` is set to 
+        `true`, you must include an `Authorization` request header with your 
+        [JSON Web Token](https://tools.ietf.org/html/rfc7519) (JWT) as the 
+        value. You can get a token using `POST /v1/register` or `POST /v1/token`.
```

Comment:
@iamigo Actually /v1/tokens does not work. We had /v1/token endpoint until now because we started with only one token create/return.
Now we can change this endpoint to /v1/tokens, but this will touch a lot of files. So, I did not clutter this PR. I am going to create next PR with just /v1/token -> /v1/tokens changes.